### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ethan-master-coding**
### Description

Changed the link to the CC forums to the link to the CC get help subcategory.

### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Comment by ethan-master-coding** [comment]
We should probably close #621 when this is merged. And you cant link discussions.

**Comment by Dusch4593** [comment]
> We should probably close #621 when this is merged. And you cant link discussions.

You can still write "Closes #621" under "Description" and it should also do the trick 🪄

**Comment by Dusch4593** [comment]
Also, since this one has to do with editing documentation, let's bring in @sonnynomnom and @yangc95 for review and feedback. 😄

**Comment by ethan-master-coding** [comment]
@Dusch4593 looking a discussion makes it so only maintainers can replay but everybody can still view it. And it can be unlocked again.

**Comment by yangc95** [comment]
@ethan-master-coding Cool idea! I was wondering if we can even create a new topic in [Community Forum](https://discuss.codecademy.com/c/community/1406) @sonnynomnom 🤔

**Comment by ethan-master-coding** [comment]
@yangc95 topic or category. A topic is a thread and a category well the name speaks for its self

**Comment by yangc95** [comment]
> Community Forum

I see why you linked to the "Codecademy" (meta convos) category now. 

I'm concerned with discussions around Docs getting lost among the general meta convos, so maybe we can create a new category in [Get Help](https://discuss.codecademy.com/c/get-help/1801).

**Comment by ethan-master-coding** [comment]
@yangc95 Ok. So best cause scenario we have a new subcategory in [#get help](https://discuss.codecademy.com/c/get-help/1801.

**Comment by ethan-master-coding** [comment]
About possibly adding a subcategory for the docs Mabey we should add badges for # of contributions to the docs with the [GH plugin](https://www.discourse.org/plugins/github.html).

File: README.md [hunk 1]
```diff
@@ -46,4 +46,4 @@ Please read through the [Contribution Guide](https://github.com/Codecademy/docs/
 
 ## Additional Notes
 
-Remember, if you ever have any questions at all, we're always here to help in the [Codecademy Forums](https://discuss.codecademy.com/) and [Codecademy Discord](https://discord.com/invite/codecademy).
+Remember, if you ever have any questions at all, we're always here to help in the [Codecademy Forums](https://discuss.codecademy.com/c/get-help/codecademy/1836) and [Codecademy Discord](https://discord.com/invite/codecademy).
```
Comment (Dusch4593) [inline README.md:49]: For the "Codecademy Discord" link, should we mention which channel(s) to check out after their invite is accepted?

Comment (ethan-master-coding) [reply README.md:49]: I would say yes. Something like:

For general questions about the docs use `#general` but for language-specific questions use the channel for that language.

**Comment by yangc95** [comment]
> @yangc95 Ok. So best case scenario we have a new subcategory in [#get help](https://discuss.codecademy.com/c/get-help/1801).

The best-case scenario is to get a new subcategory. But revisiting the [Forum](https://discuss.codecademy.com/) and looking at the description for "Get Help" vs "Community," I can't help but think Docs should go in Community:

Screen Shot 2022-03-19 at 11 19 38 PM

Let me first see if getting a category added is possible 👍

**Comment by ethan-master-coding** [comment]
@sonnynomnom Guessing you accidentally missed this notification. Mind giving your input when you have some time.

**Comment by ethan-master-coding** [comment]
@Dusch4593 @yangc95 If we do get a category to think we should copy and paste the contribution guide and some other things to try and prevent a lot of questions like `How do I contribute?` and other questions of that nature that are answered by these files already?

File: README.md (hunk 1 above)
Comment (yangc95) [inline README.md:49]:
```suggestion
Remember, if you ever have any questions at all, we're always here to help in the [Codecademy Forums](https://discuss.codecademy.com/c/community/community-docs/2205) and [Codecademy Discord](https://discord.com/invite/codecademy).
```

**Comment by ethan-master-coding** [comment]
> Mabey we should add badges for # of contributions to the docs with the [GH plugin](https://www.discourse.org/plugins/github.html).

Think we should do this? And should we copy-paste the contribution guide and stuff to prevent duplicate questions?

**Comment by yangc95** [comment]
> 
> And we can now merge this right?

I think this description looks great for now! I'll approve of this PR 🚀  Currently, we're adding some new topics to the category for some more talking points/structure, ideas are welcome!

**Review by yangc95 — APPROVED** [review]

**Review by Dusch4593 — APPROVED** [review]
LGTM as well! 👍🏻

**Comment by ethan-master-coding** [comment]
@Dusch4593 @yangc95 think we should add some tips for new contributors to the post? Like, take on an issue for your first contribution and stuff like that.

**Pull request merged**